pkgs/by-name/hy/hyfetch/package.nix

```diff
@@ -23,6 +24,13 @@ python3Packages.buildPythonApplication rec {
     python3Packages.typing-extensions
   ];
 
+  makeWrapperArgs = [
+    "--prefix"
+    "PATH"
+    ":"
+    (lib.makeBinPath [ pciutils ])
+  ];
+
```

Comment:
Will this wrap `neowofetch`? `neowofetch` is also expected to use as a standalone script I suppose.

Reply:
Good point, it won't.